cmd/mdatagen/templates/readme.md.tmpl

```diff
@@ -19,6 +19,11 @@
 {{- end }}
 {{- if ne $class "" }}
 | Issues        | ![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3A{{ $class }}%2F{{ $shortName }}%20&label=open&color=orange&logo=opentelemetry) ![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3A{{ $class }}%2F{{ $shortName }}%20&label=closed&color=blue&logo=opentelemetry) |
+{{- end }}
+{{- if .Status.Codeowners }}
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | {{ stringsJoin .Status.Codeowners.Active ", " }} |
```

Comment:
Should these be links to the code owners?

Reply:
Not sure what you mean - to the .github/CODEOWNERS file?

Reply:
i meant to https://github.com/<username>, the @ symbol makes me think it was intended to be a link in the readme.

Reply:
Now I see, yes, let me do that.

Reply:
Done!